mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulVisitedProfileScreen.kt

```diff
@@ -82,27 +82,29 @@ fun StatefulVisitedProfileScreen(
     uid: String,
     onMatchClick: (ChessMatchAdapter) -> Unit,
     onPuzzleClick: (PuzzleInfoAdapter) -> Unit,
+    onBackToSocialClick: () -> Unit,
```

Comment:
Nitpick : name this onBack, since we don't know which screen opened this composable.

Reply:
Refactored in `[64a6544](https://github.com/epfl-SDP/android/pull/303/commits/64a6544a2aa602d5d620d6672eaab107e301e665)`